kore/src/Kore/Internal/Predicate.hs

```diff
@@ -6,98 +6,159 @@ License     : NCSA
 
 module Kore.Internal.Predicate
     ( Predicate -- Constructor not exported on purpose
-    , pattern PredicateAnd
-    , pattern PredicateFalse
-    , pattern PredicateTrue
-    , compactPredicatePredicate
+    , PredicateF
+        ( AndF
+        , BottomF
+        , CeilF
+        , EqualsF
+        , ExistsF
+        , FloorF
+        , ForallF
+        , IffF
+        , ImpliesF
+        , InF
+        , NotF
+        , OrF
+        , TopF
+        )
```

Comment:
Why do these need to be explicitly exported?

Reply:
I think these were made explicit by mistake. My IDE has a feature where if I misclick on a line it automatically explicitly imports/exports all the terms used in the file.
Fixed it now.